lib/api/controllers/documentController.js

```diff
@@ -767,25 +768,31 @@ class DocumentController extends NativeController {
   async updateByQuery (request) {
     let query = request.getBodyObject('query');
     const changes = request.getBodyObject('changes');
+    const strict = request.getBoolean('strict');
     const silent = request.getBoolean('silent');
     const userId = request.getKuid();
     const refresh = request.getString('refresh', 'false');
     const source = request.getBoolean('source');
     const { index, collection } = request.getIndexAndCollection();
     const lang = request.getLangParam();
+    const methodName = 'core:storage:public:document:updateByQuery';
```

Comment:
Captain nitpicker here but it's considered a good practice (for readability) to split the code into logical blocks separated by spaces.  

For example, here the first part of the function is only about extracting arguments from the request, then we are modifying some of those arguments if needed and finally we are calling external services.

IMHO, the `methodName` declaration should be made in another "logical block", just before being use for the first name for example.

Also, it should rather `eventName` so you can even more simplify it with simply `event`

Reply:
Separated by spaces or lines ?
I wanted to keep consistency with the whole code. Should I change `methodName` to `event` in all other methods ?